src/main/java/org/prebid/server/events/EventsService.java

```diff
@@ -37,21 +28,10 @@ public Events createEvent(String bidId, String bidder) {
     /**
      * Returns events enabled flag for the given account.
      * <p>
-     * This data is not critical, so returns false if any error occurred.
-     */
-    public Future<Boolean> isEventsEnabled(String publisherId, Timeout timeout) {
-        return applicationSettings.getAccountById(publisherId, timeout)
-                .map(account -> ObjectUtils.defaultIfNull(account.getEventsEnabled(), false))
-                .otherwise(EventsService::accountFallback);
-    }
-
-    /**
-     * Returns false if account is not found or any exception occurred.
+     * This data is not critical, so returns false if null.
      */
-    private static Boolean accountFallback(Throwable exception) {
-        if (!(exception instanceof PreBidException)) {
-            logger.warn("Error occurred while fetching account", exception);
-        }
-        return false;
+    public Boolean isEventsEnabled(Account account) {
```

Comment:
I think we don't this method anymore. We may check `EventsEnabled` directly in `EventsService`.